test/metabase/test/data/crate.clj

```diff
@@ -0,0 +1,75 @@
+(ns metabase.test.data.crate
+  "Code for creating / destroying a Crate database from a `DatabaseDefinition`."
+  (:require [environ.core :refer [env]]
+            metabase.driver.crate
+            (metabase.test.data [generic-sql :as generic]
+                                [interface :as i])
+            [metabase.driver.generic-sql :as sql]
+            [clojure.java.jdbc :as jdbc]
+            [metabase.util :as u]
+            [clojure.string :as s])
```

Comment:
Do you mind ordering the `:requires`? In Metabase we keep the the dependent namespaces ordered like:

*  Clojure namespaces
*  3rd-party namespaces
*  metabase namespaces

See other MB namespaces to get a feel for how we do it.